signs-work-order-map/utils/mapUtils.tsx

```diff
@@ -112,14 +145,34 @@ export const useCreateSignPins = (
               key={`marker-${sign.id}`}
               longitude={sign.lng}
               latitude={sign.lat}
-              color={sign.isLocationDetailPage ? "red" : undefined}
+              color={
+                sign.isLocationDetailPage
+                  ? "red"
+                  : sign.source === "agol"
+                    ? undefined // Don't use default color for AGOL signs
+                    : undefined
```

Comment:
I'm curious - is this second, inner ternary holding a place for a future enhancement? It stuck out since both branches resolve to `undefined`. If not a placeholder, can you reduce this down to just the enclosing ternary?